cider/include/cider/CiderRuntimeModule.h

```diff
@@ -60,6 +60,7 @@ class CiderRuntimeModule {
   size_t getGroupByAggHashTableBufferNum() const;
   bool isGroupBy() const;
   bool hasCountDistinct() const;
+  bool isUsingArrowFromat() const;
```

Comment:
I think we should not expose such API. User side should know what compileOption/runtimeOption is.

Reply:
Agree, I will set a private member in operator